test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -2456,7 +2456,7 @@ function MyFunction ($param1, $param2)
 
     It 'Should complete module specification keys in using module statement' {
         $res = TabExpansion2 -inputScript 'using module @{'
-        $res.CompletionMatches.CompletionText -join ' ' | Should -BeExactly "GUID MaximumVersion ModuleName ModuleVersion RequiredVersion"
+        $res.CompletionMatches.CompletionText -join ' ' | Should -BeExactly "ModuleName GUID ModuleVersion RequiredVersion MaximumVersion"
     }
 }
 
```

Comment:
now that we'll be having a `ToolTip` which is _not_ just the `CompletionText`, should we be validating that they are indeed different?

Reply:
Tooltips are different from the completiontext in many scenarios (completion of commands, command parameters, etc.) and yet we don't have tests for those so there's not really a precedent for it. Depending on how the test is written, it would also be annoying to have to update the tests if you make slight changes to the phrasing, grammar, etc. of a tooltip.  
So I don't personally think it's needed, but I can add it if requested.

Reply:
@MartinGC94 ?

Reply:
Sorry, I forgot to respond but I did modify a test to also validate that the tooltip is different from the completion text: https://github.com/PowerShell/PowerShell/pull/17864/commits/2a04d1afcf37939cc95dda67c40c66d3bd967253